en/development/testing.rst

```diff
@@ -501,6 +501,10 @@ table linked to the model called Article. You can use any model available in
 your application. The statement will only import the Article schema, and  does
 not import records. To import records you can do the following::
 
+.. versionchanged:: 2.7 
+    ``model`` has been replaced by ``table``
```

Comment:
Hi Mark

wasn't getting anything. no error and it wasn't populating my 'test' db
using the 'import' table

As soon as I changed to 'table' (instead of model) worked right away,
starting populating my table with the fixtures i included.

I'm using 2.7.2

But I didn't make my models as it was an upgrade from an earlier 2.x cake
version

I hope this helps


---
Sacha Lewis Dmytruk - President
*undoLogic inc. *
Montreal: 1.514.488.5775 ext 6
Toll-Free: 1.855.488.5775 ext 6
undoLogic.com <http://undologic.com/>

On Fri, Aug 28, 2015 at 8:21 PM, Mark Story <notifications@github.com>
wrote:

> In en/development/testing.rst
> <https://github.com/cakephp/docs/pull/3192#discussion_r38255979>:
>
> > @@ -501,6 +501,10 @@ table linked to the model called Article. You can use any model available in
> >  your application. The statement will only import the Article schema, and  does
> >  not import records. To import records you can do the following::
> >
> > +.. versionchanged:: 2.7
> > +    ``model`` has been replaced by ``table``
>
> You should still be able to use 'model'. Do you get an error when you use
> it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cakephp/docs/pull/3192/files#r38255979>.
>